schema/StatisticalVariable.mcf

```diff
@@ -179,3 +179,99 @@ medicalStatus: dcs:PatientDeceased
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+testResult: dcs:Positive
+
+Node: dcid:CumulativeCount_MedicalTest_COVID_19_Negative
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+testResult: dcs:Negative
+
+Node: dcid:Count_MedicalTest_COVID_19_Pending
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+testResult: dcs:Pending
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientRecovered
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientRecovered
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientDeceased
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_PatientHospitalized
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalStatus: dcs:PatientHospitalized
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientHospitalized
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientHospitalized
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_PatientInICU
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalStatus: dcs:PatientInICU
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientInICU
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientInICU
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_PatientOnVentilator
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalStatus: dcs:PatientOnVentilator
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientOnVentilator
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientOnVentilator
```

Comment:
We decided not to add the new StatisticalVariables to the schema folder. These are all the same as the ones in your import subdir, correct? If so, we can just undo these changes.

Reply:
Yes, let me do some changes.

Reply:
Yep, this is also what I am thinking as this will cause conflicts easily and would potentially cause some problems unintentionally.